content/docs/dvclive/live/log_artifact.md

```diff
@@ -68,6 +71,11 @@ all the metadata passed as arguments to the corresponding `dvc.yaml`. Passing
   artifact. Useful if you don't want to track the original path in your repo
   (for example, it is outside the repo or in a Git-ignored directory).
 
+- `cache` - <abbr>cache</abbr> the files with DVC to
+  [track](/doc/dvclive/how-it-works#track-large-artifacts-with-dvc) them outside
+  of Git. Defaults to `True`, but set to `False` if you want to annotate
```

Comment:
Minor and might be only my opinion, but the explanation of `False` feels redundant

Reply:
Yeah, not that strong an opinion, but since I don't think there's an equivalent in other loggers, I think it's worth being explicit here about why you would want to use this option.